server/views/pages/induction/workInterests/workInterestTypes.njk

```diff
@@ -21,39 +21,39 @@
 
 {% block content %}
 
+  {% if errors.length > 0 %}
+    {{ govukErrorSummary({
+      titleText: 'There is a problem',
+      errorList: errors,
+      attributes: { 'data-qa-errors': true }
+    }) }}
+  {% endif %}
+
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
-      {% if errors.length > 0 %}
-        {{ govukErrorSummary({
-          titleText: 'There is a problem',
-          errorList: errors,
-          attributes: { 'data-qa-errors': true }
-        }) }}
-      {% endif %}
-
       <form class="form" method="post" novalidate="">
         <div class="govuk-form-group">
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
           {% set otherHtml %}
             {{ govukTextarea({
-              id: "workInterestsOther",
-              name: "workInterestsOther",
+              id: "workInterestTypesOther",
+              name: "workInterestTypesOther",
               rows: "2",
-              value: form.workInterestsOther,
+              value: form.workInterestTypesOther,
               type: "text",
               label: {
                 text: "Give details",
                 attributes: { "aria-live": "polite" }
               },
-              attributes: { "aria-label" : "Give details of any work " + prisonerSummary.firstName + " " + prisonerSummary.lastName + " has done before" },
```

Comment:
Oh my days! Thats hilarious - clearly not much testing happened on the CIAG side! 🙄